workspaces/web/webpack.config.js

```diff
@@ -22,6 +22,12 @@ module.exports = {
         resolve: { alias, extensions, mainFields },
         module: {
             rules: [
+                {
+                    test: /\.m?js/,
```

Comment:
why are we adding this rule? Do you have some explanation?

Reply:
>  Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.00:01
 - configuration.module.rules[0].resolve has an unknown property 'fullySpecified'. These properties are valid:

This tells be that something is wrong here :thinking: